### DISCUSSION TITLE
IMage2Image bug

### DISCUSSION THREAD

**Discussion by Yrbashazar** [hlky/stable-diffusion-webui#469] [General]
Image2Image don't work i have this ![Screenshot_2022-09-01-22-30-11-049_com.google.chromeremotedesktop.jpg](https://user-images.githubusercontent.com/110055383/188006848-1c142252-cc8f-4fa1-8648-58e1129e4af0.jpg) help me please !

**Comment by codedealer** [comment]
It was fixed, update please

**Reply by Yrbashazar** [reply]
Yes thanks very much
